amy/templates/includes/workshopinquiry_details.html

```diff
@@ -29,6 +29,14 @@
         <td>
           {% include "includes/request_institution.html" with admin=admin object=object %}
         </td></tr>
+    <tr><th>Affiliated with a member?</th>
+      <td>
+        {{ object.member_affiliation|default:"&mdash;" }}
+      </td></tr>
+    <tr><th>Member registration code:</th>
+      <td>
+        {{ object.member_code|default:"&mdash;" }}
+      </td></tr>
```

Comment:
Indentation of `<td>`s is slightly off compared to `amy/templates/includes/selforganisedsubmission_details.html`.
<img width="198" alt="image" src="https://github.com/carpentries/amy/assets/72821/2db3c50e-7222-4f6b-8e96-2ba658e130c1">